L/llama_cpp/build_tarballs.jl

```diff
@@ -85,12 +92,15 @@ products = [
     ExecutableProduct("embedding", :embedding),
     ExecutableProduct("main", :main),
     ExecutableProduct("perplexity", :perplexity),
+    ExecutableProduct("q8dot", :q8dot),
     ExecutableProduct("quantize", :quantize),
+    ExecutableProduct("quantize-stats", :quantize_stats),
+    ExecutableProduct("vdot", :vdot),
     LibraryProduct("libllama", :libllama),
 ]
 
 dependencies = Dependency[
 ]
 
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6", preferred_gcc_version = v"8.1.0")
+               julia_compat="1.6", preferred_gcc_version = v"12")
```

Comment:
GCC 12 with C++ code won't work with any released version of julia, as simple as that.

Reply:
Can we bump gcc-11.1.0 to gcc-11.3.0 in 0_RootFS in Yggdrasil? Not sure how painful that is. gcc-11.1.0 will produce wrong code as mentioned in the NOTES for one of the intrinsics used.

Reply:
Actually
```console
% nm -D libllama.so | grep GLIBCXX_3.4.2 | cut -d'@' -f2|sort -u
GLIBCXX_3.4.20
GLIBCXX_3.4.21
GLIBCXX_3.4.22
GLIBCXX_3.4.29
```
So there isn't much difference between GCC 12.1 and 11.3 and apparently julia v1.6.7 has libstdc++.so.6.0.29, I haven't checked previous versions